README.md

```diff
@@ -17,24 +17,25 @@ git clone https://github.com/compSPI/LatentSpaceVisualizer.git
 Change the current working directory to the root of this repository.
 
 ```bash
-cd /path/to/LatentSpaceVisualizer
+cd LatentSpaceVisualizer
 ```
 
-Create the Python environment that has the dependencies required to run the code.
+Download from the Anaconda Cloud and install the Python environment that has the dependencies required to run the code.
 
 ```bash
-conda env create -f environment.yml
+conda env create compSPI/compSPI
 ```
 
 Activate the environment.
+
 ```bash
-conda activate latent_space_visualizer
+conda activate compSPI
 ```
 
 Install the kernel.
+
 ```bash
-python -m ipykernel install --user --name latent_space_visualizer \ 
-	--display-name "Python (latent_space_visualizer)"
+python -m ipykernel install --user --name compSPI --display-name "Python (compSPI)"
```

Comment:
The `--user` option was not understood on my Mac, so the kernel was not built. I'll try from the cluster tomorrow.